e2e/cypress/integration/channel/archived_channels_spec.js

```diff
@@ -156,4 +158,319 @@ describe('Leave an archived channel', () => {
             cy.get('#search-items-container .no-results__wrapper').should('be.visible');
         });
     });
+
+    it('MM-T1696 - When clicking "More..." for Public and Private channels, no options for archived channels are shown when the feature is disabled', () => {
+        cy.apiAdminLogin();
+        cy.apiUpdateConfig({
+            TeamSettings: {
+                ExperimentalViewArchivedChannels: false,
+            },
+        });
+
+        // # Create public channel
+        cy.apiCreateChannel(testTeam.id, 'channel', 'channel').then(({channel}) => {
+            // # Visit the channel
+            cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+
+            // # Archive the channel
+            cy.uiArchiveChannel();
+
+            // # Open more modal
+            cy.get('#publicChannelList').should('be.visible').within(() => {
+                cy.findByText('More...').scrollIntoView().should('be.visible').click();
+                cy.get('#channelsMoreDropdown').should('not.be.visible');
+
+                // # Modal should not contain the created channel
+                cy.get('#moreChannelsList').should('not.contain', channel.name);
+            });
+        });
+    });
+
+    it('MM-T1697 - More... for Public channels shows archived channels option', () => {
+        cy.apiAdminLogin();
+        cy.apiUpdateConfig({
+            ServiceSettings: {
+                ExperimentalChannelSidebarOrganization: 'default_off',
+            },
+            TeamSettings: {
+                ExperimentalViewArchivedChannels: true,
+            },
+        });
+
+        // # Create public channel
+        cy.apiCreateChannel(testTeam.id, 'channel', 'channel').then(({channel}) => {
+            // # Visit the channel
+            cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+
+            // # Archive the channel
+            cy.uiArchiveChannel();
+
+            // # Open more modal
+            cy.get('#publicChannelList').should('be.visible').within(() => {
+                cy.findByText('More...').scrollIntoView().should('be.visible').click();
+            });
+            cy.get('#moreChannelsModal').should('be.visible').within(() => {
+                // # Click on dropdown
+                cy.findByText('Show: Public Channels').should('be.visible').click();
+
+                // # Click archived channels
+                cy.findByText('Archived Channels').click();
+
+                // # Modal should contain created channel
+                cy.get('#moreChannelsList').should('contain', channel.display_name);
+            });
+        });
+    });
+
+    it('MM-T1698 - More... for Private channels shows archived channels option', () => {
+        cy.apiAdminLogin();
+        cy.apiUpdateConfig({
+            ServiceSettings: {
+                ExperimentalChannelSidebarOrganization: 'default_off',
```

Comment:
In hindsight, the more I see us doing this the more I wonder if we should just change the test procedure to use the new sidebar instead. What do you think/how hard is that to do?

Reply:
The ticket has tests for both new and old sidebar, and for example wants to test old sidebar specific features like channel grouping 🤷‍♀️  If we are getting rid of the old sidebar soon then these test cases will be redundant then too but for now I guess we are keeping them?

Reply:
That's true, though I'm wondering what the value is in writing these tests for the old sidebar. Especially considering that the main purpose for the push in writing these tests is for cloud, which will never be using the old sidebar. Let's chat about this in triage.

Reply:
Agreed.